### DISCUSSION TITLE
Daily 04/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#203] [Daily Sprint]
- Qué hice ayer?
- Qué haré hoy?
- Dificultades e impedimentos

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve con la página de corrección de datos de registración. Por la noche nos juntamos virtualmente para ver que cosas corregir de la página, vimos de acomodar la barra de navegación, el footer y de pasar los botones para seleccionar productos a la página de catálogo.

Hoy voy a estar haciendo las correcciones de la barra de navegación y el footer y dándole estilo a la página de contacto para que quede similar a los demás formularios. Además trataré de hacer la tarea de inglés. Por la tarde tengo clase de programación.

**Comment by BraianTroncoso** [comment]
Por la noche nos juntamos virtualmente para ver que cosas corregir de la página, vimos de acomodar la barra de navegación, el footer y de pasar los botones para seleccionar productos a la página de catálogo.

Hoy voy a modificar un poco catálogo cargandole el boton del carrito y dejando una img incluida para tener un poco mas de vision.
Y voy a re-diseñar el carrito para que se pueda visualizar unos productos ya cargados.

**Comment by Brian-Almada** [comment]
Ayer estuve viendo más de Angular. Hoy se me fue el día entre el médico y el trabajo.

**Comment by rpgrca** [comment]
Acabo de terminar un dashboard inicial para ver los productos. Mañana a la tarde planeo expanderlo un poco más para también crear los tipos de productos y los tipos de envío, también quiero ver de ajustar la base de datos porque nuestro artículo ahí no tiene título ni cantidadDisponible todavía.
